java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -2286,4 +2325,66 @@ void testIsFloat() {
       assertColumnsAreEqual(expectedDoubles, doubles);
     }
   }
+  @Test
+  void testCreateDurationDays() {
+    Integer[] days = {100, 10, 23, 1, -1, 0, Integer.MAX_VALUE, null, Integer.MIN_VALUE};
+
+    try (ColumnVector durationDays = ColumnVector.durationDaysFromBoxedInts(days)) {
+      HostColumnVector hc = durationDays.copyToHost();
+      for (int i = 0; i < 7; i++) {
+        assertEquals(days[i], hc.getInt(i));
```

Comment:
`getInt` is supposed to assert when trying to access a `null` value.  How is it working here?

Reply:
Good catch, I had hard-coded the length so we weren't checking the entire length of the array.